skimage/__init__.py

```diff
@@ -131,51 +131,7 @@ def _test(doctest=False, verbose=False):
 doctest_verbose.__doc__ = doctest.__doc__
 
 
-# Logic for checking for improper install and importing while in the source
-# tree when package has not been installed inplace.
-# Code adapted from scikit-learn's __check_build module.
-_INPLACE_MSG = """
-It appears that you are importing a local scikit-image source tree. For
-this, you need to have an inplace install. Maybe you are in the source
-directory and you need to try from another location."""
-
-_STANDARD_MSG = """
-Your install of scikit-image appears to be broken.
-Try re-installing the package following the instructions at:
-http://scikit-image.org/docs/stable/install.html """
-
-
-def _raise_build_error(e):
-    # Raise a comprehensible error
-    local_dir = osp.split(__file__)[0]
-    msg = _STANDARD_MSG
-    if local_dir == "skimage":
-        # Picking up the local install: this will work only if the
-        # install is an 'inplace build'
-        msg = _INPLACE_MSG
-    raise ImportError("""%s
-It seems that scikit-image has not been built correctly.
-%s""" % (e, msg))
```

Comment:
I don't think it's wise to get rid of this completely. These messages were added so that users trying to run skimage from source without compiling didn't get an opaque import error like `no module _foo_cy`.

Reply:
Hmm ok, maybe we should refractor this more like [scikit-learn's](https://github.com/scikit-learn/scikit-learn/tree/master/sklearn/__check_build) which has it in a fully seperate directory that compiles a specific dummy pyx file.

Reply:
So am I allowed to copy [90% of scikit-learn's code for this check](https://github.com/scikit-learn/scikit-learn/tree/master/sklearn/__check_build)? Would I have to include their "COPYING" notice in the specific folder?

They use the [New BSD License](https://github.com/scikit-learn/scikit-learn/blob/master/COPYING)
```
  a. Redistributions of source code must retain the above copyright notice,
this list of conditions and the following disclaimer.
```